toke.c

```diff
@@ -4458,9 +4468,11 @@ S_intuit_more(pTHX_ char *s, char *e)
     if (!send)		/* has to be an expression */
         return TRUE;
 
-    /* this is terrifying, and it works */
+    /* this is terrifying, and it mostly works.  See GH #16478 */
 
     int weight;
+
+    /* Unsigned versions of current character, and the penultimate one */
```

Comment:
s/penultimate/previous/

I'd expect "the penultimate one" to be the last-but-one character of the string.

Reply:
You're right